client/app/components/composites/SearchBar/SearchBar.js

```diff
@@ -57,11 +57,17 @@ class SearchBar extends Component {
 
     // Bind `this` within methods
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleResize = this.handleResize.bind(this);
 
-    this.state = { selectedPlace: null };
+    this.state = {
+      selectedPlace: null,
+      mobileMenu: false,
```

Comment:
Oh, ok! I thought it was related to a different menu layout on mobile, so maybe name change is good 👍